pages/mesosphere/dcos/2.1/monitoring/logging/aggregating/elk/index.md

```diff
@@ -11,144 +11,84 @@ enterprise: false
 
 
 
-You can pipe system and application logs from the nodes in a DC/OS cluster to an Elasticsearch server. These instructions are based on CentOS 7 and might differ substantially from other Linux distributions.
```

Comment:
Why are you removing this part? don't you think this is helping to avoid missunderstandings?

Reply:
It was removed in https://github.com/mesosphere/dcos-docs-site/pull/2628/files#diff-caaad885fa59f8bc0ded9916e4c2de34L14